x-pack/test/functional/es_archives/saved_objects_management/feature_controls/security/data.json

```diff
@@ -66,20 +66,3 @@
     }
   }
 }
-
-{
```

Comment:
saved objects management sorts by `type` and the API doesn't allow sorting by two fields. So having a 6 config object and a 8 config object causes unstable tests because of the unstable ordering.